packages/@react-spectrum/s2/src/bar-utils.ts

```diff
@@ -64,15 +65,7 @@ export const track = () => ({
   borderRadius: 'full',
   backgroundColor: {
     default: 'gray-300',
-    staticColor: {
-      white: {
-        default: 'transparent-white-100'
-      },
-      // TODO: Is there a black static color in S2?
-      black: {
-        default: 'transparent-black-400'
-      }
-    },
+    isStaticColor: 'transparent-overlay-300',
```

Comment:
just to verify, this value is essentially the transparent white 300 that the figma token references for the progress bar right? And we'd ditched the black color token because it doesn't have an actual design token definition?

Reply:
it's either transparent-white-400 or transparent-black-400 depending on the background color. According to the tokens figma file they updated it to be consistent.